doc/conf.py

```diff
@@ -84,7 +85,10 @@ def _check_deps():
 autosummary_generate = True
 
 autodoc_docstring_signature = True
-autodoc_default_flags = ['members', 'undoc-members']
+if sphinx.version_info < (1, 8):
+    autodoc_default_flags = ['members', 'undoc-members']
+else:
+    autodoc_default_options = {'members': '', 'undoc-members': ''}
```

Comment:
As far as I understand the values should be None and not empty string. 
https://github.com/sphinx-doc/sphinx/pull/5315/files#r211091546

Reply:
'' appears to work too and having None mean 'on' is pretty terrible IMO (the sphinx maintainer seems to agree :)) but sure, I changed it as that's what's documented.